L1Trigger/L1CaloTrigger/interface/Phase2L1CaloEGammaUtils.h

```diff
@@ -1245,17 +1247,20 @@ namespace p2eg {
      * unique to each GCT card.
      */
     l1tp2::DigitizedClusterCorrelator createDigitizedClusterCorrelator(const int corrTowPhiOffset) const {
+      std::cout << " Creating digitized cluster correlator with towEta, crEta: " << towEta << ", " << crEta
+                << ", with global crystal iEta " << (towEta * CRYSTALS_IN_TOWER_ETA) + crEta << std::endl;
```

Comment:
@skkwan Could you remove this in our fork too? Then this can be merged, because https://github.com/cms-sw/cmssw/pull/44390 was merged too.

Reply:
Thanks for pointing this out, done in commit [8a906f8](https://github.com/cms-l1t-offline/cmssw/pull/1223/commits/8a906f837f42bc60897fafcb9c9caf948fe7b129)